src/main/java/org/recordrobotics/charger/subsystems/Drive.java

```diff
@@ -30,20 +36,40 @@ public class Drive extends SubsystemBase {
 
 	private DifferentialDrive _differentialDrive = new DifferentialDrive(_leftMotors, _rightMotors);
 
-	private static final FeedbackDevice SELECTED_SENSOR = FeedbackDevice.CTRE_MagEncoder_Relative;
+	private TalonFXSensorCollection[] _leftCollections = {
+		new TalonFXSensorCollection(new BaseTalon(RobotMap.DriveBase.LEFT_FRONT_MOTOR_PORT, "left1")),
+		new TalonFXSensorCollection(new BaseTalon(RobotMap.DriveBase.LEFT_MIDDLE_MOTOR_PORT, "left2")),
+		new TalonFXSensorCollection(new BaseTalon(RobotMap.DriveBase.LEFT_BACK_MOTOR_PORT, "left3"))
+	};
+	private TalonFXSensorCollection[] _rightCollections = {
+		new TalonFXSensorCollection(new BaseTalon(RobotMap.DriveBase.RIGHT_FRONT_MOTOR_PORT, "right1")),
+		new TalonFXSensorCollection(new BaseTalon(RobotMap.DriveBase.RIGHT_MIDDLE_MOTOR_PORT, "right2")),
+		new TalonFXSensorCollection(new BaseTalon(RobotMap.DriveBase.RIGHT_BACK_MOTOR_PORT, "right3"))
+	};
 
 	private static final double GEAR_RATIO = 10.75;
 	private static final double WHEEL_DIAMETER = 6 * 25.4; // diameter in inches * conversion rate to millimeters
 
+	private GenericEntry _encoderEntry;
+
 	public Drive() {
 		_leftMotors.set(0);
 		_rightMotors.set(0);
-		_left[0].configSelectedFeedbackSensor(SELECTED_SENSOR, 0, 10);
-		_left[1].configSelectedFeedbackSensor(SELECTED_SENSOR, 0, 10);
-		_left[2].configSelectedFeedbackSensor(SELECTED_SENSOR, 0, 10);
-		_right[0].configSelectedFeedbackSensor(SELECTED_SENSOR, 0, 10);
-		_right[1].configSelectedFeedbackSensor(SELECTED_SENSOR, 0, 10);
-		_right[2].configSelectedFeedbackSensor(SELECTED_SENSOR, 0, 10);
```

Comment:
Probably should return the selected sensor constant

Reply:
wdym

Reply:
`SELECTED_SENSOR`

Reply:
yes, but why should we return it, and to where?

Reply:
No, just how it was before at top of class.
Reason to return it is because you are using the same value for each sensor and it would be easier to switch if needed to any reason.

Reply:
o wait, misread
sorry, gotcha